src/index.ts

```diff
@@ -216,6 +221,7 @@ export class FileSystem {
     );   
   }
 
+  // localPath: path to write the data to on local machine
```

Comment:
/**
*In remote mode, write data to file in host machine.
*@param localPath: destination file path in host machine.
*@param data: data to write into file.
*/

Reply:
Fixed.